assets/src/edit-story/elements/text/display.js

```diff
@@ -103,10 +123,24 @@ function TextDisplay({
       : '';
   });
 
+  if (backgroundType === BACKGROUND_TEXT_MODE.HIGHLIGHT) {
+    return (
+      <HighlightElement ref={ref} {...props}>
```

Comment:
It's really underseriable to change refs. Any way this can be done differently? Some top-level container? Or we can apply the exact same DOM structure of `p > span`, but customize the CSS for both `p and span` when a specific background mode is used.

Reply:
Interesting. I will try this `top-level container` approach for this case. Thanks, @dvoytenko !

Reply:
I did not address this one, though. Is it necessary?

Reply:
@dvoytenko @barklund what's the resolution here? Dima, can you advise on whether this should still be addressed before merging?